### PR TITLE
373898 tagger

### PR DESCRIPTION
**Pull request by veejayricana**
Current Description Fix:
- Update the tagger files to tags-placeholder
- Add a key value for each industry or subject
- Restructure the tags.xlsx and refactor the code accordingly instead of using the placeholders

Old Description Fix: 
- Update tagger display text base on placeholder - for translation
- Sanitize for tagger actual value (data-title) 
- Add condition for edge case (Strategy & Consulting)

Test URLs:
- Before: https://main--accenture-newsroom--hlxsites.hlx.page/tools/tagger/index.html
- After: https://373898-tagger--accenture-newsroom--hlxsites.hlx.page/tools/tagger/index.html

### CODE REVIEW

**Comment by rpapani** [comment]
@veejayricana why do we need to use place holders? Every geo can have their own copy of tags.xlsx file and use their own values. The tagger will open the geo version of the tag.xlsx to show the values.

**Comment by ravkiran** [comment]
@veejayricana why do we need to use place holders? Every geo can have their own copy of tags.xlsx file and use their own values. The tagger will open the geo version of the tag.xlsx to show the values.

**Comment by veejayricana** [comment]
Hi @ravkiran reason some of the tagging has its own translation. here is the example:
image
Specially in local geo.

**Comment by ravkiran** [comment]
@jjjaspher looks like the requirement is to have a different name and values for geo tags. In that case, I'd propose to change the format of the tagger implementation to use 
- separate sheets for industries and subjects
- in each sheet, we can add key and value. Key is the one author will see in the tagger Interface and value is the one that'll be copied when the author clicks on the copy action in tagger

I don't want to mix the code between author tooling (like tagging and library etc..) and consumer experience. Placeholders we'll only use it for doing localization in the end user website, not for the authors tooling.

@amol-anand WDYT?

**Comment by jjjaspher** [comment]
Hi @amol-anand  @ravkiran, we created a new tags json "tags-placeholders.xlsx" with new structure to accommodate key value as a solution for the different translation or text each geo. We are planning to create a new tags-placeholders.xlsx instead of modifying the existing tags.xlsx for the reason of we need to create first all new tags-placeholders.xlsx across geo before merging this code so that there will be no downtime of tagger tool. Please review the code changes and let us know if you have concern. Thank you!
CC: @veejayricana

File: tools/tagger/tagger.js [hunk 1]
```diff
@@ -1,55 +1,65 @@
...
     html += '</div>';
   });
+
   const results = document.getElementById('results');
   results.innerHTML = html;
 }
 
 async function getTaxonomy() {
-  const resp = await fetch('/tags.json');
+  const resp = await fetch('/tags-placeholders.json');
```
Comment (rpapani) [inline tools/tagger/tagger.js:49]: can we avoid `placeholders` name? how about `new-tags` ?

File: tools/tagger/tagger.js [hunk 2]
```diff
@@ -1,55 +1,65 @@
...
-        <span data-title="${tag}" class="tag cat-${catId % 8}">${tag}</span>
+        <span data-title="${tag.key}" class="tag cat-${catId % 8}">${tag.text}</span>
       </span>
     `;
     }
   });
   return html;
 }
 
+const getReduceTags = (oTaxonomy, sCategory) => {
```
Comment (rpapani) [inline tools/tagger/tagger.js:16]: any reason why prefixing variables with `o` and `s`?

**Comment by rpapani** [comment]
@veejayricana can you add the tags-placeholders.xlsx in en, so that we can validate https://373898-tagger--accenture-newsroom--hlxsites.hlx.page/tools/tagger/index.html ?

File: tools/tagger/tagger.js (hunk 2 above)
Comment (jjjaspher) [reply tools/tagger/tagger.js:16]: @ravkiran that means the data type of the variable, o = Object, s =String. Just a habit of mine. Please let us know if we need to update it. Thanks!

File: tools/tagger/tagger.js (hunk 1 above)
Comment (veejayricana) [reply tools/tagger/tagger.js]: @ravkiran done renaming the files to new-tags.

cc: @jjjaspher

**Comment by veejayricana** [comment]
> @veejayricana can you add the tags-placeholders.xlsx in en, so that we can validate https://373898-tagger--accenture-newsroom--hlxsites.hlx.page/tools/tagger/index.html ?

@ravkiran Done moving the files to EN folder and it is ready for verification.

cc: @jjjaspher

**Comment by veejayricana** [comment]
> @veejayricana looking at the structure
> 
> image
> It is typically intuitive for authors to see the text first and then the value rather than the key.

@ravkiran Done restructuring.
image

**Review by rpapani — APPROVED** [review]

**Comment by rpapani** [comment]
@veejayricana Looks good. Please feel free to merge when you're ready.
